packages/e2e-tests/specs/editor/blocks/classic.test.js

```diff
@@ -98,4 +100,43 @@ describe( 'Classic', () => {
 		await page.waitForSelector( '.wp-block[data-type="core/gallery"]' );
 		expect( await getEditedPostContent() ).toMatch( /<!-- wp:gallery/ );
 	} );
+
+	it( 'Should not fail after save/reload', async () => {
+		const mainBlockCSSSelector = 'div[aria-label^="Block: Classic"]';
+
+		const clickClassic = async () => {
+			await page.waitForSelector( mainBlockCSSSelector );
+			await page.click( mainBlockCSSSelector );
+		};
+
+		// Might move to utils if this becomes useful enough for other tests
+		const runWithoutCache = async ( cb ) => {
+			await page.setCacheEnabled( false );
+			await cb();
+			await page.setCacheEnabled( true );
+		};
```

Comment:
TBH I was considering just inlining this, since it's really just two (fairly intuitive) commands. Might not make much sense to encapsulate them.

Reply:
We need to make sure that `page.setCacheEnabled( true )` runs at the end even if the first bits throw.

Reply:
Ah, good point!

Reply:
Ah yes, good point indeed!

Reply:
> TBH I was considering just inlining this

Yeah, fine with me, we don't really use it anywhere else so you have a point.

Reply:
Besides the actual method to disable and enable the cache are self-descriptive, the additional helper function doesn't add much in terms of meaning.